### PR TITLE
Document supported table options

### PR DESCRIPTION
**Pull request by adduc**

### CODE REVIEW

File: docs/migrations.rst [hunk 1]
```diff
@@ -464,6 +464,38 @@ To simply change the name of the primary key, we need to override the default ``
...
+========= ===========
+collation set collation of table (defaults to utf8_general_ci)
+comment   set a text comment on table
+engine    set engine (defaults to InnoDB)
+========= ===========
+
+For the PostgreSQL adapter:
+
+======= ===========
+Option    Description
```
Comment (rquadling) [inline docs/migrations.rst]: A couple of spaces.

Comment (adduc) [inline docs/migrations.rst]: Thanks, I've fixed the spacing. Are there any additional changes needed?

File: docs/migrations.rst [hunk 2]
```diff
@@ -464,6 +464,38 @@ To simply change the name of the primary key, we need to override the default ``
...
+id          set name of automatically created identity field
+primary_key set field to use for primary key (defaults to value of id)
+=========== ===========
+
+For the MySQL adapter:
+
+========= ===========
+Option    Description
+========= ===========
+collation set collation of table (defaults to utf8_general_ci)
```
Comment (adduc) [inline docs/migrations.rst]: Yes, charset is also available (I'll add it in a subsequent commit).

By config overrides, are you referring to setting charset/collation in the configuration file?

Comment (adduc) [inline docs/migrations.rst]: Looking into [`Phinx\Db\Adapter\MysqlAdapter::createTable`](https://github.com/robmorgan/phinx/blob/0.6.x-dev/src/Phinx/Db/Adapter/MysqlAdapter.php#L243), it doesn't look like charset is supported as a table option (collation is used to determine character set).

**Comment by dereuromark** [comment]
ping @adduc 
Anything missing or can we merge?

**Comment by adduc** [comment]
I think so, although I'm not sure if there's been any changes to supported table options since 0.6.

**Comment by dereuromark** [comment]
I see. But even with it being not fully complete this would already be an improvement here.

**Comment by dereuromark** [comment]
@lorenzo Can we already merge this partial docs upgrade?

**Pull request merged**